data/human/news.txt

```diff
@@ -72,6 +72,11 @@ news "merchant anywhere"
 			"Having both feet firmly on the ground doesn't make you take one step to completing your missions."
 			"The darkest hour is just before the departure."
 			"Don't look a gift shuttle in the engine outlets."
+			"Getting there is half the fun. Getting paid is the other half."
+			"Being a pilot is not all fun and games. We have angry customers, hostile pirates and asteroids to deal with."
```

Comment:
I'm not sure what somebody is saying in the world of Endless Sky would necessarily be contraction-free. It sounds too formal for someone you're randomly meeting once and will never interact with again.
```suggestion
			"Being a pilot isn't all fun and games. We have angry customers, hostile pirates and asteroids to deal with."
```

Reply:
I'm not a native English speaker, so I sometimes forget contractions. It was not my intention to make those merchants sound very formal.

Reply:
(It also wasn't my intention to make myself sound very formal.)